doc/getting_started/index.rst

```diff
@@ -38,6 +38,11 @@ itself.
 Bootstrap west
```

Comment:
This is for line 28  / the whole section but github doesn't let me comment there.
> Zephyr's multi-purpose `west` tool lets you easily get all the source code in one command instead of cloning multiple repos one by one. It also simplifies configuration and provides convenience wrappers like "west flash" that abstract away hardware differences. It is possible not to use `west` if you prefer to use your own tools or need to troubleshoot the tools `west` relies on, however this requires manual / extra steps / effort and expert knowledge.

Reply:
That's all true and good knowledge, but I am trying to avoid making the getting started guide longer than it is. Hope you agree that's worth doing. I'm going to meld that into the main west index.rst since that's linked to from here -- please let me know how it looks to you once that's up.

Reply:
The text above is a (much shorter) *replacement*, not an addition :-)

Reply:
> The text above is a (much shorter) replacement for both paragraphs; not an addition :-)

This is longer from what I can see, so let me be explicit about what I think you are saying, and you can correct my misunderstanding.

You originally wrote:

> This is for line 28 / the whole section

The current section around line 28 is:

```
Get the source code
*******************

Zephyr's multi-purpose :ref:`west` tool lets you easily get the Zephyr project
source code, instead of manually cloning the Zephyr repos along with west
itself.

.. note::
   If you prefer to manage the repositories manually or with another tool
   instead of using ``west``, you can still opt to do so. The documentation
   section :ref:`no-west` describes how to work with Zephyr without using
   west to manage its source code.
```

This is your suggestion:

```
Zephyr's multi-purpose west tool lets you easily get all the source code in one command instead of cloning multiple repos one by one. It also simplifies configuration and provides convenience wrappers like "west flash" that abstract away hardware differences. It is possible not to use west if you prefer to use your own tools or need to troubleshoot the tools west relies on, however this requires manual / extra steps / effort and expert knowledge.
```

This is the same section with this PR:

```
Get the source code
*******************

Zephyr's multi-purpose :ref:`west` tool lets you easily get the Zephyr project
source code, instead of manually cloning the Zephyr repos along with west
itself.

.. warning::

   It's possible to use Zephyr without installing west, but you have
   to **really** know :ref:`what you are doing <no-west>`.
```

This PR's current text is shorter.

Reply:
It wasn't "much shorter" sorry. It felt shorter because it was all packed in a single section and not spread across a .note and a .warning around line 40.
Forget it, I trust you (even more so after trying to compare force pushes in github :-)